src/closure_decorator.rs

```diff
@@ -159,11 +146,21 @@ impl<'a> VisitMut for ClosureDecorator<'a> {
     // prepend the `register` and `bind` function declarations
     // and the `register` calls into the top of every module.
     prepend_stmts(items, new_stmts.into_iter());
+
+    items.push(ModuleItem::Stmt(Stmt::Expr(ExprStmt {
+      expr: Box::new(Expr::Assign(AssignExpr {
+        span: DUMMY_SP,
+        op: AssignOp::Assign,
+        left: PatOrExpr::Expr(Box::new(Expr::Ident(self.stash.clone()))),
+        right: undefined_expr(),
+      })),
+      span: DUMMY_SP,
+    })))
```

Comment:
Why?

Reply:
Ah, woops. I noticed that the stash value was being serialized and tried to clear the value, but this didn't work because it would need to cleared from the nested scopes too. Will remove.